### DISCUSSION TITLE
Implement base url

### DISCUSSION THREAD

**Discussion by GabrielCWT** [felix-berlin/astro-breadcrumbs#154] [Ideas]
Hi,

I'm currently trying to make a website which will be hosted on github pages, I was wondering if it were possible to implement a base url feature so that the breadcrumbs work with github pages.

Much appreciated!

**Comment by AndrMalet** [comment]
Hi i want to help

**Reply by felix-berlin** [reply]
If you like, you are welcome to create a PR for this feature 😊

**Comment by felix-berlin** [comment]
@GabrielCWT sorry for the late response. 

Just so I understand you correctly, instead of a relative URL like `/en/category` you want to specify the full domain as an optional property to e.g.: `https://test.dev/en/category`?

**Reply by GabrielCWT** [reply]
For example the homepage for the bread crumb can be foo.com/bar instead of foo.com and every bread crumb after would be foo.com/bar/baz

Home > baz

**Reply by felix-berlin** [reply]
@GabrielCWT v.1.8.0 is out and comes with baseUrl support --> https://github.com/felix-berlin/astro-breadcrumbs/releases/tag/v1.8.0

**Reply by GabrielCWT** [reply]
Thank you so much!

**Reply by GabrielCWT** [reply]
@felix-berlin when using the breadcrumb with baseUrl, the breadcrumbs produced all have a href of baseUrl. 

Home > project > project_name
All link to baseurl.com/foo

Maybe just to clarify, for my website, I want relative url paths which include a base path.

Home (/foo) > project(/foo/project) > project_name(/foo/portfolio/project_name)

Sorry for any misunderstandings caused

**Reply by felix-berlin** [reply]
@GabrielCWT I probably misunderstood something initially, but I'm not sure at the moment. Can you check again yourself whether the latest updates have solved the problem?
